crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3055,6 +3053,173 @@ impl Ranged for crate::nodes::ParameterWithDefault {
     }
 }
 
+/// An expression that may be parenthesized.
+#[derive(Clone, Debug)]
+pub struct ParenthesizedExpr {
```

Comment:
Can we make this type private (or pub(super) at most) and move it to `parser.rs` because it is mainly an internal workaround, but by no means public api

Reply:
Unfortunately no -- it somehow becomes part of the public API in the generated LALRPOP code.